packages/node/src/store.ts

```diff
@@ -85,9 +82,16 @@ export class Store {
   public async getMultisigAddressFromAppInstance(
     appInstanceId: string
   ): Promise<string> {
-    return this.storeService.get(
-      `${this.storeKeyPrefix}/${DB_NAMESPACE_APP_INSTANCE_ID_TO_MULTISIG_ADDRESS}/${appInstanceId}`
-    );
+    for (const sc of (await this.getStateChannelsMap()).values()) {
+      if (
+        sc.proposedAppInstances.has(appInstanceId) ||
+        sc.appInstances.has(appInstanceId) ||
+        (sc.hasFreeBalance && sc.freeBalance.identityHash === appInstanceId)
+      ) {
+        return sc.multisigAddress;
+      }
+    }
+    return ""; // FIXME
```

Comment:
Don't return empty string here - throw instead if there's no multisig address to be found for some app instance ID.